src/components/SystemSettingsProvider.js

```diff
@@ -56,3 +60,15 @@ SystemSettingsProvider.propTypes = {
 export default SystemSettingsProvider
 
 export const useSystemSettings = () => useContext(SystemSettingsCtx)
+export const useHasSupersetSupport = () => {
```

Comment:
`useSupersetIsSupported` perhaps?

Reply:
Thanks for flagging this up. I think use `is` over `has` is preferable here as you suggest. In the end I went with `useIsSupersetSupported`, so that - when using this hook - a nice consistent `is`-prefixed boolean can be used:
```js
const isSupersetSupported = useIsSupersetSupported()
```